docs/source/ResultFormats.rst

```diff
@@ -88,6 +88,11 @@ output in the standard output of the runner, simply use::
 
 .. note:: The dash `-` in the option `--xunit`, it means that the xunit result
           should go to the standard output.
+.. note:: In case your tests produce unnecessary too long outputs, you can
```

Comment:
AFAICT, as a non-native speaker, "too unnecessary long outputs" doesn't sound correct.  Suggestion would be "unnecessary output that is too long".

Reply:
sure, thanks